modules/installation-creating-aws-control-plane.adoc

```diff
@@ -168,6 +168,12 @@ the CloudFormation template for the security group and roles.
 * `m5a.8xlarge`
 * `m5a.12xlarge`
 * `m5a.16xlarge`
+* `m6i.xlarge`
```

Comment:
there are no limitations in UPI templates anymore, https://github.com/openshift/installer/pull/5166

Reply:
Here was also updated.